src/main/resources/application.properties

```
@@ -0,0 +1,9 @@
+# DataSource Configuration
+spring.datasource.url=jdbc:postgresql://localhost:5432/cmsDb
+spring.datasource.username=postgres
+spring.datasource.password=password
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+# Hibernate Configuration
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=create
```

```suggestion
spring.jpa.hibernate.ddl-auto=none
```
Disable autometic table creation